content/2020-06-15-rust-ecosystem-overview/index.md

```diff
@@ -154,6 +154,9 @@ It would be impossible to list all the companies that use Rust. This is a sub-se
     *   [Android uses Rust](https://android-review.googlesource.com/q/rust)
     *   [Chromium is beginning to adopt Rust](https://chromium-review.googlesource.com/q/project:experimental/chromium/src+branch:refs/wip/rust-experimental-branch)
     *   Google released an “unofficial” project called `[tarpc](https://github.com/google/tarpc)` “An RPC framework for Rust”
+*   Linkerd
```

Comment:
I tried to stick to well-known companies. Personally, I don't know this company so I'm going to refrain from adding it.